test/helpers/verification.spec.ts

```diff
@@ -1,11 +1,12 @@
+import type { NetlifyPluginOptions, NetlifyPluginUtils } from '@netlify/build'
```

Comment:
All the changes below are just eslint auto-fixing things and prettier kicking in as well.